.gitignore

```diff
@@ -113,6 +113,7 @@ ENV/
 .floo
 .flooignore
 
+tests/data
```

Comment:
why is this in gitignore?

Reply:
The autotune tests leave some log files there

Reply:
I can change where they're logged